client/state/stats/lists/utils.js

```diff
@@ -437,7 +437,7 @@ export const normalizers = {
 
 			// ’ in country names causes google's geo viz to break
 			return {
-				label: country.country_full.replace( /’/, "'" ),
+				label: viewData.location || country.country_full.replace( /’/, "'" ),
```

Comment:
The required change was so small that I didn't make a `statsCountryViews` equivalent, like `statsLocationViews`, for now. Maybe this will be required when plotting the regions/cities in the geochart map, but let's do it when/if required.